civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -34,7 +34,7 @@
   $statusesCount[0] ++;
 endforeach;
 ?>
-<div data-ta-documents ng-controller="myController" class="chr_table-w-filters chr_table-w-filters--documents row">
+<div data-ta-documents ng-controller="ModalController" class="chr_table-w-filters chr_table-w-filters--documents row">
```

Comment:
It's newly  created `ModalController` in `civihr_employee_portal/js/ta-documents-app.js`